Readme.md

```diff
@@ -2,7 +2,7 @@
 
 ## 1. Information about this fork
 
-This is a fork of the [official PLCrashReporter repository](https://github.com/plausiblelabs/plcrashreporter) that is maintained by the [App Center](https://appcenter.ms) team. It is based on PLCrashReporter 1.2.1 (commit fda233062b5586f5d01cc527af643168665226c0) with additional fixes and changes.
+This is a fork of the [official PLCrashReporter repository](https://github.com/plausiblelabs/plcrashreporter) and is maintained by the [App Center](https://appcenter.ms) team. It is based on PLCrashReporter 1.2.1 (commit fda233062b5586f5d01cc527af643168665226c0) with additional fixes and changes.
```

Comment:
"This is a fork of the [official PLCrashReporter repository](https://github.com/plausiblelabs/plcrashreporter. This fork is maintained..." maybe?